src/libraries/System.Security.Cryptography.Cose/src/Resources/Strings.resx

```diff
@@ -157,7 +157,7 @@
     <value>Critical Headers must be a definite-length CBOR array of at least one element.</value>
   </data>
   <data name="DecodeCoseSignatureMustBeArrayOfThree" xml:space="preserve">
-    <value>COSE Signature must be a definite-length array of 3 elements.</value>
+    <value>COSE Signature must be an array of 3 elements.</value>
```

Comment:
```suggestion
    <value>COSE Signature must be an array of three elements.</value>
```

Since DecodeSign1ArrayLengthMustBeFour wrote out the word, and I think there's some sort of guideline somewhere that numbers less than 10 should use words instead of digits.